components/kyma-environment-broker/cmd/broker/broker_suite_test.go

```diff
@@ -182,9 +179,7 @@ func NewBrokerSuiteTest(t *testing.T, version ...string) *BrokerSuiteTest {
 	provisioningQueue.SpeedUp(10000)
 	provisionManager.SpeedUp(10000)
 
-	scheme := runtime.NewScheme()
-	apiextensionsv1.AddToScheme(scheme)
-	fakeK8sSKRClient := fake.NewClientBuilder().WithScheme(scheme).Build()
+	fakeK8sSKRClient := fake.NewClientBuilder().WithScheme(sch).Build()
```

Comment:
there is no need for multiple `scheme` in tests, the broker also uses the same `scheme` for all operation queues